packages/core/core/test/AssetGraphBuilder.test.js

```diff
@@ -55,35 +54,12 @@ describe('AssetGraphBuilder', () => {
 
   it('creates an AssetGraphBuilder', async () => {
     invariant(
-      builder.graph.nodes.has(
+      builder.assetGraph.nodes.has(
         new Dependency({
           moduleSpecifier: './module-b',
           env: DEFAULT_ENV
         }).id
       )
     );
   });
-
-  it('throws a BuildAbortError when resolving if signal aborts', async () => {
```

Comment:
I've removed the `AbortController` code from `AssetGraphBuilder` because we have plans to move it up to Parcel. I plan on adding that back in a separate PR.